eng/Generation.psm1

```diff
@@ -71,16 +75,16 @@ function Invoke-Cadl($baseOutput, $projectName, $sharedSource="", $fast="", $deb
         Push-Location $repoRootPath
         Try
         {
-            # node node_modules\@cadl-lang\compiler\dist\core\cli.js compile --output-path $outputPath "$baseOutput\$projectName.cadl" --emit @azure-tools/cadl-csharp
-            $emitCommand = "node node_modules/@cadl-lang/compiler/dist/core/cli.js compile --output-path $outputPath $baseOutput/$projectName.cadl --emit @azure-tools/cadl-csharp"
+            $cadlFileName = $mainFile ? $mainFile : "$baseOutput/$projectName.cadl"
```

Comment:
why do we need this default name?

Reply:
Default name refers to? Cadl-ranch cadl will be passed by $mainFile, TestProjects cadl will be passed by "$baseOutput/$projectName.cadl".